src/controllers/organization.controller.js

```diff
@@ -235,3 +235,45 @@ export const unsubscribeToOrganization = async (req, res) => {
     }
   }
 };
+
+export const resyncOrganization = async (req, res) => {
+  let transaction;
+  try {
+    await assertIfReadOnlyMode();
+    await assertDataLayerAvailable();
```

Comment:
take out assertDataLayerAvailable(); and assertWalletIsAvailable(); because @frantzarty added middle ware to automatically check these. Its in his next PR